app/models/user.rb

```diff
@@ -1,8 +1,11 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Let's make sure the application asks for confirmation of email. you can add(`:confirmable`)

Reply:
Thanks @uwadonat for the feedback, let me add it quickly.